tritium-registry/src/main/java/com/palantir/tritium/metrics/registry/SlidingWindowTaggedMetricRegistry.java

```diff
@@ -21,18 +21,8 @@
 
 public final class SlidingWindowTaggedMetricRegistry extends AbstractTaggedMetricRegistry {
 
-    private static final TaggedMetricRegistry DEFAULT = new SlidingWindowTaggedMetricRegistry(1, TimeUnit.MINUTES);
-
     public SlidingWindowTaggedMetricRegistry(int window, TimeUnit windowUnit) {
         super(() -> new SlidingTimeWindowArrayReservoir(window, windowUnit));
     }
 
-    /**
-     * Get the global default {@link TaggedMetricRegistry}.
-     */
-    @SuppressWarnings("unused") // public API
-    public static TaggedMetricRegistry getDefault() {
```

Comment:
Deleting this is a compile break, but the method was only introduced in tritium 0.11.10 (released two days ago), so if we assume people haven't started compiling against this in the last two days then this should be fine...

Reply:
I would also consider the switch for default tagged registry from exponential decay to sliding window makes this a `break` as well, so will make this `0.12.0`